### PR TITLE
User story short version

### PR DESCRIPTION
**Pull request by arbalisi12**

### CODE REVIEW

File: cit3.0-web/src/components/Page/UserStory/UserStory.js [hunk 1]
```diff
@@ -120,47 +144,73 @@ export default function UserStory() {
...
+      setIsOkButton(false);
+      redirectUrl.current = param.url;
     } else {
-      setIsYesButton(false);
-      setIsNoButton(false);
+      setIsGoButton(false);
+      setIsOkButton(false);
     }
+
     if (userOption.code.includes("-NO")) {
```
Comment (jjstratton) [inline cit3.0-web/src/components/Page/UserStory/UserStory.js:200]: should this be removed?

File: cit3.0-web/src/components/Page/UserStory/UserStory.js [hunk 2]
```diff
@@ -120,47 +144,73 @@ export default function UserStory() {
...
-      setIsNoButton(true);
-      setIsYesButton(false);
+      const isNo = userOption.user_story_paths.find((x) =>
```
Comment (jjstratton) [inline cit3.0-web/src/components/Page/UserStory/UserStory.js:201]: Also should these No's be yes' or go's?

File: cit3.0-web/src/components/Page/UserStory/UserStory.js (hunk 1 above)
Comment (arbalisi12) [reply cit3.0-web/src/components/Page/UserStory/UserStory.js:200]: so, if NO and the next path is also NO, it means end of path. There is a GENERIC-NO path which leads to END.
some NO answers will go to another story path.
and yeah, there is a long and version version where YES and NO have a different behavior.

File: cit3.0-web/src/components/Page/UserStory/UserStory.js (hunk 2 above)
Comment (arbalisi12) [reply cit3.0-web/src/components/Page/UserStory/UserStory.js:201]: it depends on the rules of the path, some requires YES and NO, some goes directly to Lets GO

**Review by jjstratton — APPROVED** [review]

**Pull request merged**